README.md

```diff
@@ -1,4 +1,82 @@
 # terraform-aws-databricks-workspace-custom-vpc
-Terraform module for creating a Databricks workspace in AWS within an existing VPC
 
-[Full module reference](docs/reference.md)
\ No newline at end of file
+Nextail Terraform module for creating a Databricks workspace in AWS within an existing VPC
+
+<!-- BEGIN_TF_DOCS -->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.1.2 |
+| <a name="requirement_aws"></a> [aws](#requirement\_aws) | >= 3.37.0 |
+| <a name="requirement_databricks"></a> [databricks](#requirement\_databricks) | >= 0.5.7 |
```

Comment:
Isn't this the version we want to move forward?

Reply:
Yes, but the change is not on main (yet). The intention of the PR, if approved, is to use this new command in the other branch to update the other PR docs

Reply:
updated!